tools/hermetic_grunt/Dockerfile

```diff
@@ -0,0 +1,54 @@
+# To build and run the hermetic grunt:
+# docker build -f tools/hermetic_grunt/Dockerfile -t uproxy-grunt . && \
+# docker run --rm -v $(pwd):/root/repository_root -it uproxy-grunt
+#
+# You have to run from the repository root. The output is written to the out/
+# directory.
+
+FROM ubuntu
+
+LABEL description="Docker container for hermetic grunt runs"
+
+###############################################################################
+# Install prerequisite global tools
+###############################################################################
+
+RUN apt-get update
+
+RUN apt-get install -qqy git python build-essential
+
+# Add a fresher repository for nodejs
+RUN apt-get install curl && \
```

Comment:
I had issues initially, but they can now be all installed in one go. I've also removed the need for explicitly installing curl. Docker's ADD can fetch things from the internet.